Management/src/ActuatorApi/README.md

```diff
@@ -0,0 +1,111 @@
+    ```
+
+    *Optional* - Add forecast data starting from a specific date and/or number of days:
+
+    ```shell
+    dotnet run --runtask=ForecastWeather --fromDate=10/18/2024 --days=30
+    ```
+
+> [!NOTE]  
+> For the `fromDate` parameter, use values formatted as `yyyy-dd-MM` or `MM/dd/yyyy`.
+
+> [!TIP]
+> The task [`ResetTask`](./AdminTasks/ResetTask.cs) can be used to remove all forecast data:
+> `dotnet run --runtask=ResetWeather`
+
+1. Use your preferred IDE or `dotnet run` to start the application.
+
+## Running on Tanzu Platform for Cloud Foundry
+
+1. Create a MySQL service instance in an org/space
+
+   ```shell
+   cf target -o your-org -s your-space
+   ```
+
+    * When using VMware MySQL for Tanzu Application Service:
+
+      ```shell
+      cf create-service p.mysql db-small sampleMySqlService
+      ```
+
+    * When using the Cloud Service Broker for Azure:
+
+      ```shell
+      cf create-service csb-azure-mysql small sampleMySqlService
+      ```
+
+    * When using the Cloud Service Broker for GCP:
+
+      ```shell
+      cf create-service csb-google-mysql your-plan sampleMySqlService
+      ```
+
+1. Wait for the service to become ready (you can check with `cf services`)
+1. Run the `cf push` command to deploy from source (you can monitor logs with `cf logs actuator-api-management-sample`)
+    * When deploying to Windows, binaries must be built locally before push. Use the following commands instead:
+
+      ```shell
+      dotnet publish -r win-x64 --self-contained
+      cf push -f manifest-windows.yml -p bin/Release/net8.0/win-x64/publish
+      ```
+
+1. Copy the value of `routes` in the output and open in your browser. The app should start and respond to requests, but the database still needs to be configured with the tasks listed in the next section.
+
+> [!NOTE]  
+> The provided manifest will create an app named `actuator-api-management-sample` and attempt to bind it to the MySql service `sampleMySqlService`.
+
+### Running Tasks
+
+Depending on the steps taken to push the application to Cloud Foundry, the commands below may require customization (for example, if the application was not published before pushing to a Linux cell, the path for the command might be `./bin/Debug/net8.0/linux-x64/Steeltoe.Samples.ActuatorApi`)
+
+1. Apply Entity Framework Core database migration scripts:
+
+    ```shell
+    cf run-task actuator-api-management-sample --command "./Steeltoe.Samples.ActuatorApi runtask=MigrateDatabase" 
+    ```
+
+1. Run [`ForecastTask`](./AdminTasks/ForecastTask.cs) to predict weather for the next 7 days:
+
+    ```shell
+    cf run-task actuator-api-management-sample --command "./Steeltoe.Samples.ActuatorApi runtask=ForecastWeather" 
+    ```
+
+> [!TIP]  
+> To remove all forecast data, run [`ResetTask`](./AdminTasks/ResetTask.cs):
+>
+> ```shell
+> cf run-task actuator-api-management-sample --command "./Steeltoe.Samples.ActuatorApi runtask=ResetWeather" 
+> ```
```

Comment:
This indent is wrong now.

Reply:
This is a limitation in GitHub markdown, documented at https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts:

> Alerts cannot be nested within other elements.